tests/std/tests/P0896R4_ranges_subrange/test.cpp

```diff
@@ -113,671 +84,671 @@ namespace test_view_interface {
         STATIC_ASSERT(ranges::range<V>);
         STATIC_ASSERT(!ranges::range<V const>);
         STATIC_ASSERT(ranges::view<V>);
-        STATIC_ASSERT(!CanEmpty<V>);
-        STATIC_ASSERT(!CanEmpty<V const>);
-        STATIC_ASSERT(!CanBool<V>);
-        STATIC_ASSERT(!CanBool<V const>);
-        STATIC_ASSERT(!CanData<V>);
-        STATIC_ASSERT(!CanData<V const>);
-        STATIC_ASSERT(!CanSize<V>);
-        STATIC_ASSERT(!CanSize<V const>);
-        STATIC_ASSERT(!CanFront<V>);
-        STATIC_ASSERT(!CanFront<V const>);
-        STATIC_ASSERT(!CanBack<V>);
-        STATIC_ASSERT(!CanBack<V const>);
-        STATIC_ASSERT(!CanIndex<V>);
-        STATIC_ASSERT(!CanIndex<V const>);
+        STATIC_ASSERT(!CanEmpty<V&>);

```

Comment:
Ugh - sorry for this enormous set of mechanical changes. This is `s/CanFront/CanMemberFront/g`, `s/CanBack/CanMemberBack/g`, and tacking `&` onto every type argument since the "old" concepts in this file didn't care about value category but the "new" concepts in `<range_algorithms_support.hpp>` do.